pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -86,9 +86,11 @@ def test_searchsorted(self):
         # Searching for single item argument, side='left' (default)
         res_cat = c1.searchsorted('apple')
         assert res_cat == 2
+        assert tm.is_scalar(res_cat)
```

Comment:
``tm.is_scalar`` guards against res_cat being an array, as ``assert np.array(...) == 2`` would always pass the assertion test, as long as the array is non-empty.

Another way would be to to do ``assert (res_cat == 2) is True``.